students/rachel_s/session02/series.py

```diff
@@ -0,0 +1,54 @@
+def fibonacci(n):
+    """Return the nth value in the fibonacci series starting with 0 index"""
+    if n <= 1:
+        return n
+    else:
+        return fibonacci(n-1) + fibonacci(n-2)
+
+
+def lucas(n):
+    """Return the nth value in the Lucas numbers series starting with 0 index"""
+    if n == 0:
+        return 2
+    elif n == 1:
+        return 1
+    else:
+        return lucas(n-1) + lucas(n-2)
+
+
+def sum_series(n, n0=0, n1=1):
```

Comment:
It is great to see the effort you put in in going thru all the warm-ups!
So, at this point something to think about: fibonacci, lucas and sum_series are series summation functions. sum_series is a generalized form, while fibonacci and lucas are special cases with certain initial values.
Question: is it necessary to maintain a code base with series summation logic in three places?

Reply:
I think actually the intention is not to rewrite `fibonacci()` and `lucas()` with `sum_series()`, but to derive `sum_series()` from the two similar `fibonacci()` and `lucas()`, so I think there's no need to rewrite. Yes, we saw another student's submission that rewrote `fibonacci()` and `lucas()`, andI thought that's OK as well (but not necessary for the sake of this assignment).